team-telnyx/webrtc
# feat(WEBRTC-121): Added and configured ESlint and Husky

- Added and configured ESlint and Husky

## 📝 To Do


## ✋ Manual testing

1. run `npm run build`
2. check if the build was generated successfully

## 🦊 Browser testing

### Desktop
